### ISSUE TITLE
Error: Cannot find module ‘/var/www/html/vuestorefront/vue-storefront/dist/vue-ssr-client-manifest.json’

### ISSUE THREAD

**Issue by cristian-g** [DivanteLtd/vue-storefront#2662]
Using “yarn build” and then “yarn start” causes an error in pm2 as you can see in the attached screenshot.

Error: Cannot find module ‘/var/www/html/vuestorefront/vue-storefront/dist/vue-ssr-client-manifest.json’

![78b45483a07e14c133053841b27edb5b8d4aff03](https://user-images.githubusercontent.com/15870631/55072301-053beb80-508b-11e9-9957-14e80e6a3efb.png)

I’ve read that VueSSRClientPlugin generates vue-ssr-client-manifest.json in the output directory during the build, but in my case the build is not generating that file. I am using the default build configuration. What could be wrong?

Thank you all!

**Comment by pkarw** [comment]
Hmm, that's strange. Have You tried `yarn pm2 reload all` after `yarn build`?

**Comment by cristian-g** [comment]
I solved this by doing the compilation in Mac OS instead of doing it in Linux. If I compile on Mac OS, it does generate the json. Maybe it was a permissions issue on some system folder, but I do not know. I’ll keep investigating.

**Comment by alexryall** [comment]
I have the same error. Only files generated on linux in the dist folder are:
core-service-worker.js  index.amp.html  index.basic.html  index.html  index.minimal.html  service-worker.js

On mac a lot more files are generated such as dist/vue-ssr-client-manifest.json and the server starts with errors.

**Comment by benkelly92** [comment]
I was also having this issue and believe it's related to memory. 

We use a combination of Mac, Linux and WSL. On our Mac & WSL machines, it doesn't happen. On our Linux boxes, it would happen occasionally, and we'd need to rebuild to fix it.

Today we tried to spin up a testing server, using DigitalOcean's cheapest droplet and we'd get this issue with every build. However, after upping the memory from 1GB to 2GB, we could get it to work. So this leaves me to believe that it is not a Linux issue, but a memory issue. This makes sense because our WSL builds should run the same software (Ubuntu 18.04), but would always succeed, however, these machines have more memory. 

TLDR; Using at least 2GB or more of memory seems to fix this.

**Comment by alanboy** [comment]
@benkelly92 This was the key for me. I was building inside a VM with 3GB and the build process died leaving missing bundles in the dist folder. I had to increase my VM memory to 5GB (I have other processes running).

**Comment by simonmaass** [comment]
we are having this exact same isse - but on a 8GB linux machine... this error is really strange... if we manually re-trigger the 'yarn build' command it sometimes works and sometimes not... has anyone found any new insights into what is exactly happening? maybe also why the command doesnt return an error but the build finished as "success" eventhou the file is missing...

**Comment by simonmaass** [comment]
is there any reason that this plugin is used
- import VueSSRPlugin from 'vue-ssr-webpack-plugin';

instead of
- import VueSSRServerPlugin from 'vue-server-renderer/server-plugin'
@pkarw
